### DISCUSSION TITLE
Recognize text from an image that has text in multiple languages

### DISCUSSION THREAD

**Discussion by ashokparmar29** [PaddlePaddle/PaddleOCR#13440] [Q&A] [answered]
Hi Everyone,
Consider an image with texts in multiple languages. I want to ask, is it possible with PaddleOCR to recognize the text from the image and show it in the image in the respective languages?

**Comment by GreatV** [comment]
PaddleOCR can't recognize multiple languages at the same time, it can only handle them separately.

**Reply by ashokparmar29** [reply]
Hi @GreatV 
Thanks for the response. I have seen this image in the readme file of the repo. It looks like PaddleOCR has the capability to recognize multiple languages at the same time. I want to find if it is correct and if yes then how to do this.
![image](https://github.com/user-attachments/assets/3a45cd92-a606-40a1-bd99-ff3bc811ed99)

**Reply by GreatV** [reply, marked as answer]
This only supports Chinese and English, if you want to support more languages at the same time, you need to expand the character dictionary and collect a lot of data to retrain.

**Reply by ashokparmar29** [reply]
Do we need to retrain the detection and recognition model or retraining the recognition model only will work? Also, which model do you think will be best for multilanguage text detection and recognition?

**Reply by ashokparmar29** [reply]
@GreatV May you give suggestions to my last query?
